spec/models/contact_form_spec.rb

```diff
@@ -0,0 +1,95 @@
+
+
+#  `rails generate curation_concerns:work GenericWork`
+
+require 'rails_helper'
+
+describe ContactForm do
+
+  describe "it requires input" do
+
+    before do
+
+      subject.contact_method = "Contact Menthod"
+      subject.category = "Category"
+      subject.name = "John Smith"
+      subject.email = "abc@123.com"
+      subject.subject = "This is the subject"
+      subject.message = "This is a message."
+
+    end
+
+
+    it "validates contact_method" do
+
+      subject.contact_method = nil
+      expect(subject).to be_valid
```

Comment:
This is a test for what functionality?  Do you want to test that `contact_method` is not used or irrelevant?